### ISSUE TITLE
Traceback if input dataframe index is numerical

### ISSUE THREAD

**Issue by Suicidal-Insanity** [InTRA-USP/IntraSOM#23]
Simple workaround is creating a copy of the input dataframe and converting the index to strings, but it would be nice if this was not needed.

Otherwise when trying to generate a u matrix image with labels or running the rep_sample() function you get a traceback complaining about concatenation of an integer and a string. (Visualization.py line 622 / intrasom.py line 2113)

**Comment by gjdv** [comment]
Agree. I encountered the same issue. I think the following replacement should do the job, but I'm just a simple user and not aware of depencies.

`value = ', '.join(value)`

becomes

`value = ', '.join([str(v) for v in value])`
